AsyncMux/Sources/MuxDB.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  MuxDB.swift
+//  AsyncMux
+//
+//  Created by Lissy Harrison on 6/5/23.
+//
+
+import Foundation
+import SQLite3
+
+class MuxDB {
+    static let shared = MuxDB()
+    
+    let SQLITE_TRANSIENT = unsafeBitCast(-1, to: sqlite3_destructor_type.self)
+    
+    // specifies to store database in Files app on user's device
+    private var dbPath = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)[0] as String
+    
+    private var db: OpaquePointer?
+    
+    private init() {
+        db = openDatabase()
+        createTable()
+    }
+    // create database
+    func openDatabase() -> OpaquePointer? {
```

Comment:
This and `createTable()` should be private as they shouldn't be used from outside.